expenses-manager-frontend/src/main/frontend/src/app/topbar/topbar.component.html

```diff
@@ -14,8 +14,8 @@ <h1>{{reloadService.getCurrentCurrency() + totalAmount}}</h1>
           <app-expense-table class="expenses-tab" selectedTab="{{tab}}"></app-expense-table>
         </ng-container>
         <ng-container *ngSwitchCase="'/reports'">
-          <h1>Reports work!</h1>
+          <app-reports-overview selectedTab="{{tab}}"></app-reports-overview>
```

Comment:
Recomand sa pui asta in app.module.ts (independent de topbar) si sa te folosesti de routes ca sa afisezi doar unde trebuie.
Pentru schimbarea in functie de tab, recomand sa te uiti la ce am facut la relatia dintre topbar (tab) si right-sidebar folosind `ReloadService`, mai exact unde sunt folosite `reloadExpenses() / reloadExpenses$` si `changeTab() / tabChange$`, s-ar putea sa iti fie folositoare. Asa elimini si acel "ngClass" de la right-sidebar.